src/components/views/HowItWorks/HowItWorks.module.css

```
@@ -0,0 +1,155 @@
+    display: inline-block;
+    font-family: var(--font-dm-mono), monospace;
+    font-size: 0.72rem;
+    font-weight: 500;
+    letter-spacing: 0.14em;
+    text-transform: uppercase;
+    color: var(--amber);
+    margin-bottom: 1.25rem;
+}
+
+.title {
+    font-family: var(--font-instrument-serif), "Georgia", serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)